torch_glow/tests/nodes/quantized_add_test.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+
+from tests.utils import jitVsGlow
+
+
+def test_quantization_basic():
+    """Basic test of the PyTorch quantize and dequantize Node on Glow."""
+
+    # This only test if operator has been successfully projected
+    # Calculating process in glow will be optimized
+    def test_simple(a, b):
+        q = torch.nn.quantized.Quantize(0.3, 0, torch.quint8)
+        dq = torch.nn.quantized.DeQuantize()
+        return dq(torch.ops.quantized.add(q(a), q(b), scale=1.0, zero_point=0))
```

Comment:
maybe a different scale and zp would be good to test here too?

Reply:
agree!